packages/datadog-plugin-bunyan/test/integration-test/client.spec.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const {
+  FakeAgent,
+  createSandbox,
+  spawnPluginIntegrationTestProc,
+  skipUnsupportedNodeVersions
+} = require('../../../../integration-tests/helpers')
+const { expect } = require('chai')
+
+const describe = skipUnsupportedNodeVersions()
+
+describe('esm', () => {
+  let agent
+  let proc
+  let sandbox
+
+  before(async function () {
+    this.timeout(20000)
+    sandbox = await createSandbox(['bunyan'], false, [`./packages/datadog-plugin-bunyan/test/integration-test/*`])
+  })
+
+  after(async () => {
+    await sandbox.remove()
+  })
+
+  beforeEach(async () => {
+    agent = await new FakeAgent().start()
+  })
+
+  afterEach(async () => {
+    proc && proc.kill()
+    await agent.stop()
+  })
+
+  context('bunyan', () => {
+    it('is instrumented', async () => {
+      proc = await spawnPluginIntegrationTestProc(
+        sandbox.folder,
+        'server.mjs',
+        agent.port,
+        (data) => {
+          const jsonObject = JSON.parse(data.toString())
+          expect(jsonObject).to.have.property('dd')
```

Comment:
It's probably worth it to add a call to `done()` here in the callback after the final expect. The `done` value can be accepted like so:

```js
    it('is instrumented', async (done) => {
```

This is a convention to help tell mocha when a function is complete and is useful when mixing async and callback patterns like what's going on here.

Reply:
can you do async (done)? i get the error:

Error: Resolution method is overspecified. Specify a callback *or* return a Promise; not both